src/Factory/UserFactory.php

```diff
@@ -86,7 +86,6 @@ protected function from_user_object($obj): CoreInterface
         ));
     }
 
-    // @todo return a UserCollection instance?
```

Comment:
Not at the moment. We discussed somewhere that user collections or collections for other types of content are not needed at the moment (https://github.com/timber/timber/issues/1689). We can add it if we really need it.